worknenjoy/gitpay
# Feat/public private status

## Description

> Displays Bounty, Private or Public statuses as Icons.

## Issue

> Closes #552 

## Impacted Area

> Task page

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user
- Click Explore Tasks
- Open a task and you should see the changes

## Before
> Screenshot from the state before

## After
> 
![image](https://user-images.githubusercontent.com/52590791/78348869-5b0d7280-75c0-11ea-858b-c9beb22076e3.png)
